app/models/ticket.rb

```diff
@@ -1158,14 +1167,13 @@ def check_title
   end
 
   def check_defaults
-    if !owner_id
-      self.owner_id = 1
-    end
+    self.owner_id = 1 unless owner_id
     return true if !customer_id
     customer = User.find_by(id: customer_id)
     return true if !customer
-    return true if organization_id == customer.organization_id
-    self.organization_id = customer.organization_id
+    return true if (customer.organization_ids.include?(organization_id) || customer.organization_id? && customer.organization_id == organization_id)
```

Comment:
Please split this line/if statment if possible. It carries way to much complexity.